widget/discrepancy-note/dn-widget.js

```diff
@@ -561,11 +561,14 @@ class Comment extends Widget {
         } );
 
         // Scroll directly to Query modal if not enough height for user to view the Query modal
-        const queryModal = this.linkedQuestion.querySelector( '.widget.or-comment-widget' );
-        if ( window.innerHeight - this.linkedQuestion.offsetHeight < queryModal.offsetHeight ) {
-            queryModal.scrollIntoView( { behavior: 'smooth', block: 'start', inline: 'nearest' } );
+        // https://github.com/OpenClinica/enketo-express-oc/issues/378
+        if ( this.linkedQuestion.classList.contains( 'or-appearance-image-map' ) ) {
+            const that = this;
```

Comment:
little nitpicky, but would be nice to not use `that` and change timeout to an arrow function